client/components/happychat/style.scss

```diff
@@ -284,10 +284,25 @@
 	padding: 0 8px;
 	margin: 6px 0;
 
+	a {
```

Comment:
Hmm, how so? If somebody writes a link, we are the ones who control how the link gets rendered; as a `a` element with this and that props and class names. Or do I get this wrong?